src/LCT.PackageIdentifier/MavenProcessor.cs

```diff
@@ -148,13 +155,13 @@ private static void SetPropertiesforBOM(ref List<Component> componentsToBOM, Com
             }
         }
 
-        private static bool ComponentPropertyCheck(Component component)
+        private static bool ComponentPropertyCheck(Component component, string constant)
```

Comment:
Add this Method in Common library ,Since it can get used by many other package type readings

Reply:
I added this method to common library